crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -39,14 +40,8 @@ pub struct TimeSeriesSpaceViewState {
     /// State of egui_plot's auto bounds before the user started dragging the time cursor.
     saved_auto_bounds: egui::Vec2b,
 
-    /// State of egui_plot's bounds.
-    saved_y_axis_range: [f64; 2],
-
-    /// To track when the range has been edited.
-    last_y_range: Option<Range1D>,
-
-    /// To track when the range lock has been enabled/disabled.
-    last_y_lock_range_during_zoom: bool,
+    /// The range of the scalar values currently on screen.
+    scalar_range: Range1D,
```

Comment:
Would be nice to state if this is the X or Y range.

Edit: looked at the video again, and now I finally understand that _in this context_ scalar (axis) meaning is as opposed to temporal (axis). I got confused because the _entire_ thing is also about scalars. Wonder if there is something to improve here? 🤔  Value range? Magnitude range?

Reply:
uhm.... I don't disagree necessarily but while I don't feel strongly about the name itself, I do feel strongly about not wanting to open a naming discussion here about some decision someone made at some point.
Plz open an issue if you think this is worth refactoring/renaming